### PR TITLE
Date ranges filter

### PR DESCRIPTION
**Pull request by stefanorosanelli**
This PR adds a first version of `date_ranges` based filter

### CODE REVIEW

File: src/Template/Element/FilterBox/filter_box_common.twig [hunk 1]
```diff
@@ -66,12 +66,21 @@
...
+        <span v-if="filter.name === 'date_ranges'">
+            <label>{{ __('From date') }}
+            <input-dynamic-attributes :value.sync="queryFilter.filter['date_ranges']['from_date']" :attrs="filter" />
+            </label>
+            <label>{{ __('To date') }}
+            <input-dynamic-attributes :value.sync="queryFilter.filter['date_ranges']['to_date']" :attrs="filter" />
+            </label>
+        </span>
+
+        <span v-if="filter.date && (filter.name !== 'date_ranges')">
```
Comment (DarioSacco) [inline src/Template/Element/FilterBox/filter_box_common.twig]: maybe this can be shortened with `v-else-if="filter.date"`

Comment (stefanorosanelli) [reply src/Template/Element/FilterBox/filter_box_common.twig]: you're right, but we should probably apply `v-else-if` to previous  `<span>` as well and add a final `v-else` ?

Comment (DarioSacco) [reply src/Template/Element/FilterBox/filter_box_common.twig]: yes `v-else-if` is ok on the second span checking `filter.type`'s value. final `v-else` is ok if the value of `filter.date` isn't relevant to show its content

Comment (stefanorosanelli) [reply src/Template/Element/FilterBox/filter_box_common.twig]: ok, have look now

**Pull request merged**